config/configmap/inferenceservice.yaml

```diff
@@ -23,6 +23,8 @@ data:
         "cpuRequest": "100m",
         "cpuLimit": "1",
         "storageSpecSecretName": "storage-config",
+        "caBundleSecretName": "",
```

Comment:
@danilopeixoto I guess this secret is required to be created for each namespace where the inference service pod is running right?